cypress/integration/core_opensearch_dashboards/opensearch_dashboards/apps/explore/04/traces_details.spec.js

```diff
@@ -0,0 +1,633 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  DATASOURCE_NAME,
+  TRACE_INDEX_PATTERN,
+  TRACE_TIME_FIELD,
+  TRACE_INDEX,
+} from '../../../../../../utils/apps/explore/constants';
+import { getRandomizedWorkspaceName } from '../../../../../../utils/apps/explore/shared';
+import { prepareTestSuite } from '../../../../../../utils/helpers';
+
+const workspaceName = getRandomizedWorkspaceName();
+
+const traceDetailsTestSuite = () => {
```

Comment:
how come we add these tests into the traces discover test file?

all the setup seems similar but will add more resource consumption

Reply:
This is the only way to get the environment set-up that i am aware of. My understanding is that we want each test suite isolated so they need to make their workspace / add data and then run the test.

Traces discover is navigated to through the Discover:Trace page so there is small over-lap but this is required to have the url for the Trace:Details page to run all of its test.

Reply:
As discussed combined testing files to traces.spec.js